src/ui/browserAction/popupPage.js

```diff
@@ -0,0 +1,294 @@
+   *                                    - Contains an HTML Event with Optinal
+   *                                    - { detail.city : ServerCity }
+   * @returns {HTMLElement} - An already rendered Element, ready to put into the DOM
+   */
+  static createServerList(list = [], onResult = () => {}) {
+    const viewElement = document.createElement("section");
+    render(
+      html`
+        <vpn-titlebar title="Select Location">
+          <img
+            slot="left"
+            src="../../assets/img/arrow-icon-left.svg"
+            @click=${onResult}
+          />
+        </vpn-titlebar>
+        <server-list .serverList=${list} @selectedCityChanged=${onResult}>
+        </server-list>
+      `,
+      viewElement
+    );
+    return viewElement;
+  }
+
+  static styles = css`
+    ${fontSizing}${resetSizing}
+    section {
+      background-color: var(--panel-bg-color);
+    }
+
+    main {
+      padding: var(--padding-default);
+    }
+    * {
+      color: var(--text-color-primary);
+      font-family: var(--font-family);
+    }
+    .row {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+    }
+    .row p {
+      flex: 1;
+      flex-grow: 1;
+    }
+    .row img:first-child {
+      margin-right: var(--padding-default);
+    }
+    .row img:last-of-type {
+      margin-left: var(--padding-default);
+    }
+    #selectPageLocation {
+      transition: all 0.15s ease;
+      background: transparent;
+      border-radius: 10px;
+      padding: calc(var(--padding-default) / 2) 0px;
+      position: relative;
+    }
+    #selectPageLocation:hover {
+      cursor: default;
+    }
+    #selectPageLocation:hover::before {
+      content: " ";
+      opacity: 0.9;
+    }
+    #selectPageLocation::before {
+      content: " ";
+      z-index: -1;
+      opacity: 0;
+      background: lch(from var(--panel-bg-color) calc(l - 15) c h);
+      transform: all 0.5s ease;
+      background: white;
+      width: 105%;
+      height: 105%;
+      border-radius: 5px;
+      position: absolute;
+      top: -2.5%;
+      left: -2.5%;
+    }
+
+    vpn-card {
+      display: block;
+      margin-bottom: calc(var(--padding-default) * 1);
+    }
+    h1,
+    h2,
+    h3 {
+      margin-top: calc(var(--padding-default) / 2);
+      margin-bottom: calc(var(--padding-default) / 2);
+    }
+
+    input {
+      width: 20px;
+      height: 20px;
+      border: 1px solid var(--border-color);
+      background-color: var(--panel-bg-color);
+    }
+    input + p {
+      margin-left: var(--padding-default);
+    }
+
+    button {
+      width: 100%;
+      border-radius: 4px;
+      padding: 8px, 16px, 8px, 16px;
+      size: 16px;
+      font-weight: 600;
+      border: 1px solid var(--action-button-color);
+      color: var(--action-button-color);
+      background-color: transparent;
+      padding: 10px;
+      margin-top: var(--padding-default);
+    }
+
+    .disabled {
+      cursor: not-allowed;
```

Comment:
do we also need `pointer-events: none` ?

Reply:
Good catch